src/main/java/org/pac4j/jax/rs/annotations/Pac4JSecurity.java

```diff
@@ -0,0 +1,42 @@
+package org.pac4j.jax.rs.annotations;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.pac4j.jax.rs.filter.SecurityFilter;
+
+/**
+ * 
+ * Identify the class or method as being filtered by {@link SecurityFilter}.
+ * 
+ * @author Victor Noel - Linagora
+ *
+ */
+@Target({ ElementType.METHOD })
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+public @interface Pac4JSecurity {
+
+    /**
+     * @return value for {@link SecurityFilter#setAuthorizers(String)}
+     */
+    String[] value() default {};
```

Comment:
If we call it value, then if it's the only parameter passed to the annotation, it is not needed to specify the parameter name: contrary to the other annotations, in this case it makes sense to me because with a RESTful API, one will almost always want to specify the authorizers like this:
`@Pac4JSecurity("isAuthentified")` instead of `@Pac4JSecurity(authorizers = "isAuthentified")`.

But I change it if you prefer.